indigo/distribution.yaml

```diff
@@ -8399,6 +8399,23 @@ repositories:
       type: git
       url: https://github.com/silviomaeta/plot_util.git
       version: master
+  plotjuggler-release:
```

Comment:
Please change the name to just `plotjuggler`. The list of `packages` needs to be removed then.